setup.py

```diff
@@ -2,8 +2,42 @@
 # SPDX-License-Identifier: Apache-2.0
 
 """This package includes dependencies of the openfl project."""
-
 from setuptools import setup
+from setuptools.command.build_py import build_py
+
+
+class GRPCBuildPyCommand(build_py):
+    """Command to generate project *_pb2.py modules from proto files."""
+
+    def run(self):
+        """Build Python and GRPC modules."""
+        from subprocess import check_call
```

Comment:
What is the reason to import inside? Probably it should be moved to the top?

Reply:
Correct. I moved these import lines to the top.